exchanges/huobi/huobi_websocket.go

```diff
@@ -56,6 +60,25 @@ const (
 	rateLimit  = 20
 )
 
+var defaultSubscriptions = subscription.List{
+	{Enabled: true, Asset: asset.Spot, Channel: subscription.TickerChannel},
+	{Enabled: true, Asset: asset.Spot, Channel: subscription.CandlesChannel, Interval: kline.OneMin},
+	{Enabled: true, Asset: asset.Spot, Channel: subscription.OrderbookChannel, Levels: 0},
```

Comment:
Suggestion: Levels: 0 looks a bit weird, any chance we can get some commentary on that? Or package const for blank values? 

NOTE: Also orderbook levels don't come up in the config so it's not as obvious that you can configure it there.

Reply:
Propose we drop Levels 0 from this, and add docs to the doc template.

Reply:
Sounds good. 👍

Reply:
Fixed aef34014 and 554f69c9